subt/docker/robotika/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM robotika/subt-base:2020-05-26
+FROM robotika/subt-base:2020-09-22
```

Comment:
is it expected that line with `apt-get update` and install remained here?

Reply:
I didn't want to mix different changes together - `ros-melodic-teleop-twist-keyboard` used to be used (I think) but I am neither the one who put it there nor the one who removed its usage (I think the usage is removed but I am not sure). If @pavliik or @jisa can give me :+1: on the removal, I'll remove it.

Reply:
OK, let's merge